addons/mail/wizard/mail_wizard_invite.py

```diff
@@ -85,3 +77,20 @@ def add_followers(self):
                 self.env['bus.bus']._sendone(self.env.user.partner_id, 'mail.message/delete', {'message_ids': message.ids})
                 message.unlink()
         return {'type': 'ir.actions.act_window_close'}
+
+    def _prepare_message_values(self, document, model_name, email_from):
+        """
+            Prepares all the data needed to create a new mail.message record
```

Comment:
Docstring does not add value compared to the method name and code, would not add it ^^ .